internetarchive/openlibrary
# Instant search result box corrected

> **Description**: hotfix

Closes #780

> **Technical**: What should be noted about the implementation?
A variable instantSearchResultState was added to save the state of the search result box. On resizing the window it restores itself to the last state it was it

> **Testing**: 
1. Load OpenLibrary.org on Desktop (not a mobile device) with a browser opened to > 800px
2. Enter the text "Lord of the Rings" into the primary search box and do not press enter. You should see realtime instant results appear
3. Click outside of the primary searchbox and the realtime instant results should disappear / hide
4. Resize the browser window < 400px.
5. Resize the browser window back to > 800px
6. realtime instant results restores to its previous state
